packages/paste-core/components/input/src/DecrementButton.tsx

```diff
@@ -22,8 +24,17 @@ export const DecrementButton = React.forwardRef<HTMLButtonElement, DecrementButt
         size="reset"
         type="button"
         borderRadius="borderRadius20"
-        backgroundColor="colorBackground"
+        backgroundColor={props.variant === "inverse" ? "colorBackgroundInverseStrong" : "colorBackground"}
         marginRight="space30"
+        _focus={{
+          borderRadius: "borderRadius20",
+          borderWidth: "borderWidth10",
+          borderStyle: "solid",
+          borderColor: props.variant === "inverse" ? "colorBorderInverseStrong" : "colorBorderPrimary",
+        }}
+        borderWidth="borderWidth10"
+        borderStyle="solid"
+        borderColor="transparent"
```

Comment:
I had to add a border to the buttons and make it transparent. After changing the _focus style I saw the icons would move slightly due to the border changing for one and not the other. Before an after shown in the video:
 https://github.com/twilio-labs/paste/assets/55083528/c99f3ecd-1f9e-4b0b-93a0-98066daf7409

Reply:
ooh ok yeah i was wondering about the use of border colors here & how you got it to work.

This solution works fine, but you could also use shadows instead. We have shadow border tokens (`shadowBorderInverseStrong` and `shadowBorderPrimary`) to account for this.

I have no like...eng perspective on which solution is better, so I'll rely on @nkrantz here for that.

Reply:
ok nvm I do have an opinion. Just realized it adds an extra 1px gray border all around the buttons.

todo: Let's use the shadow border tokens instead.

Reply:
Added boxShaow instead of border and that resolved all the spacing issues. Thank you! 🙇